projects/packages/forms/src/contact-form/js/grunion-frontend.js

```diff
@@ -6,5 +6,8 @@ jQuery( function ( $ ) {
 		dateFormat,
 		constrainInput: false,
 		showOptions: { direction: 'down' },
+		onSelect: function () {
+			$( this ).focus();
```

Comment:
Two things happening here:
1. jqueryUI doesn't trigger the input's `blur` event and by focusing the user has to `blur`
2. This is also better in general because before there was a focus loss (a11y issue).